src/v2/views/MonsterCollectionOverlay/reward.tsx

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import { theme } from "./base";
+import { styled } from "src/v2/stitches.config";
+import * as ScrollAreaPrimitive from "@radix-ui/react-scroll-area";
+
+const ScrollArea = styled(ScrollAreaPrimitive.ScrollArea, {
+  backgroundImage: theme.images.reward,
+  backgroundSize: "cover",
+  backgroundOrigin: "border-box",
+  width: 1124,
+  height: 250,
+  boxSizing: "border-box",
+  paddingTop: 40,
+  paddingLeft: 40,
+  paddingRight: 40,
+  marginTop: "auto",
+});
+
+const Viewport = styled(ScrollAreaPrimitive.Viewport, {
+  width: "100%",
+  height: "100%",
+  WebkitMaskImage:
+    "-webkit-linear-gradient(0deg, transparent 0%, black 5%, black 95%, transparent 100%)",
+  "> div[style]": {
+    display: "flex !important",
+    alignItems: "strech",
+  },
+  "& > * + *": {
+    marginLeft: 20,
+  },
+});
+
+const Scrollbar = styled(ScrollAreaPrimitive.Scrollbar, {
+  display: "flex",
+  flexDirection: "column",
+  height: 4,
+  backgroundColor: "#42291f",
+  margin: "4px 10px",
+  justifyContent: "center",
+});
+
+const Thumb = styled(ScrollAreaPrimitive.Thumb, {
+  flex: 1,
+  minHeight: 8,
+  backgroundColor: "#bf8f53",
+  borderTop: "1px solid #f3b36b",
+  borderRadius: 4,
+});
+
+const Spacer = styled("div", {
+  width: 50,
+  flexShrink: 0,
+});
+
+export const RewardSheet = ({ children }: { children: React.ReactNode }) => (
+  <ScrollArea>
+    <Viewport>
+      <Spacer />
+      {children}
+      <Spacer />
+    </Viewport>
+    <Scrollbar orientation="horizontal">
+      <Thumb />
+    </Scrollbar>
+  </ScrollArea>
+);
+
+const ItemGroupStyled = styled("section", {
+  position: "relative",
+  display: "flex",
+  flexDirection: "column",
+  "& + &": {
+    marginLeft: 40,
+  },
+  "& + &::before": {
```

Comment:
A bit of a hacky way to render dividers between `ItemGroup`s to avoid the complexity of rendering dividers using React. It works quite great and we can use `ItemGroup` without worrying about something like: is this the last component? where should I put the dividers?

But although it is a clever hack indeed you could say it is quite over-engineered and we could have gone with just a bunch of `<Divider />` components.